tests/collections/asr/mixins/adapters/test_asr_adapter_mixin.py

```diff
@@ -0,0 +1,242 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+import torch
+from omegaconf import DictConfig, OmegaConf
+
+from nemo.collections.asr.models import ASRModel, EncDecCTCModel
+from nemo.collections.asr.parts.submodules import adapter_modules
+from nemo.core import NeuralModule
+from nemo.core.classes.mixins.adapter_mixins import AdapterModuleMixin
+from nemo.utils import config_utils
+
+
+@pytest.fixture()
+def model():
+    preprocessor = {'_target_': 'nemo.collections.asr.modules.AudioToMelSpectrogramPreprocessor'}
+    encoder = {
+        '_target_': 'nemo.collections.asr.modules.ConvASREncoderAdapter',
+        'feat_in': 64,
+        'activation': 'relu',
+        'conv_mask': True,
+        'jasper': [
+            {
+                'filters': 50,
+                'repeat': 1,
+                'kernel': [1],
+                'stride': [1],
+                'dilation': [1],
+                'dropout': 0.0,
+                'residual': False,
+                'separable': True,
+                'se': True,
+                'se_context_size': -1,
+            }
+        ],
+    }
+
+    decoder = {
+        '_target_': 'nemo.collections.asr.modules.ConvASRDecoder',
+        'feat_in': 50,
+        'num_classes': 28,
+        'vocabulary': [
+            ' ',
+            'a',
+            'b',
+            'c',
+            'd',
+            'e',
+            'f',
+            'g',
+            'h',
+            'i',
+            'j',
+            'k',
+            'l',
+            'm',
+            'n',
+            'o',
+            'p',
+            'q',
+            'r',
+            's',
+            't',
+            'u',
+            'v',
+            'w',
+            'x',
+            'y',
+            'z',
+            "'",
+        ],
```

Comment:
[open discussion] does the adapter now only support English? If we had plans for multiple languages, could you pls share any thoughts about a better organization of multilingual resources, like vocab for different languages? I am still learning the code base, but i guess it may be convenient if storing resources in some file, and `import` it here.